### PR TITLE
Minor fixes to error checking for pair coalescence stats

### PR DESCRIPTION
**Pull request by nspope**
Followup to #2985

### CODE REVIEW

**Comment by nspope** [comment]
Small followup ready to go in @jeromekelleher

**Review by jeromekelleher — COMMENTED** [review]
LGTM, but missing malloc failure check

File: c/tskit/trees.c [hunk 1]
```diff
@@ -9295,6 +9289,53 @@ check_quantiles(const tsk_size_t num_quantiles, const double *quantiles)
...
+{
+    int ret = 0;
+    tsk_size_t num_nodes = self->tables->nodes.num_rows;
+    const double *nodes_time = self->tables->nodes.time;
+    double last;
+    tsk_id_t i, j;
+    double *max_time = NULL;
+    double *min_time = NULL;
+    min_time = tsk_malloc(num_bins * sizeof(*min_time));
+    max_time = tsk_malloc(num_bins * sizeof(*max_time));
```
[the remark this answers is not in the document]
Comment (nspope) [reply c/tskit/trees.c:9304]: do'h!

**Comment by nspope** [comment]
Done, thanks for catching!

**Review by jeromekelleher — APPROVED** [review]

**Comment by nspope** [comment]
Hm, automerge doesn't seem to be picking up on this?

**Comment by jeromekelleher** [comment]
@benjeffery can you take a look at mergify here please? Looks like we have a bit of a backlog to get in and it would be good to have mergify working again.

**Pull request merged**